src/services/api/AuthService.ts

```diff
@@ -87,7 +74,14 @@ class AuthService extends ApiService {
   login = async (
     credentials: CredentialsLogin
   ): Promise<UserCredentialsProp> => {
-    const { data } = await this.apiClient.post(ENDPOINTS.LOGIN, credentials)
+    const { data } = await this.apiClient.post(ENDPOINTS.LOGIN, {
+      ...credentials,
+      username: credentials.email
+    })
+
+    await asyncStorageService.setItem('token', data.access)
```

Comment:
I'm wondering where is refresh token. Do we handle token refresh?

Reply:
i will add a `await asyncStorageService.setItem('refreshToken', data.refresh)` just for easier access, but in the python django boilerplate & react native boilerplate im not seeing any payloads that are using the refresh token, should i in that case just remove it?

Reply:
not going to put `await asyncStorageService.setItem('refreshToken', data.refresh)`, if its not going to be used there is no reason to store it

Reply:
We can talk about this additionally, but we definitely need a refresh token.
I won't stop merging the PR since this can be resolved later...